elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -135,10 +148,14 @@ class PfeNavigation extends PFElement {
         item.expanded = false;
         this._activeNavigationItems = this._activeNavigationItems.filter(i => i !== item);
       } else if (item.expanded && item.parent && item.parent.visible) {
-        item.parent.expanded = true; // Ensure the parent is open
-        // If the parent item doesn't exist in the active array, add it
-        if (!this._activeNavigationItems.includes(item.parent)) {
-          this._activeNavigationItems.push(item.parent);
+        // if the parent is the mobile menu item and the size of the window is within
+        // the main breakpoint, make sure that the mobile menu is expanded
+        if (item.parent === this._menuItem && window.innerWidth <= this.breakpoints.main[1]) {
```

Comment:
We should look to refactor this in the future with matchMedia. I think it would clean up a lot of the breakpoint logic.

Reply:
created issue:
https://github.com/patternfly/patternfly-elements/issues/678